doc/nrf/ug_nrf5340.rst

```diff
@@ -0,0 +1,159 @@
+.. _ug_nrf5340:
+
+Working with nRF5340
+####################
+
+The |NCS| provides support for developing on the nRF5340 System on Chip (SoC) using the nRF5340 PDK (PCA10095).
+
+.. warning::
+   nRF5340 support is a work in progress.
+   This user guide explains how to work with nRF5340 once all required features have been merged.
+   Currently, not everything that is described here is actually supported.
+
+Introduction
+************
+
+nRF5340 is a wireless ultra-low power multicore System on Chip (SoC) with two fully programmable Arm Cortex-M33 processors: a network core and an application core.
+The |NCS| supports Bluetooth Low Energy communication on the nRF5340 SoC.
+
+See the `nRF5340 Product Specification`_ for more information about the nRF5340 SoC.
+nrf5340_dk_nrf5340 in the Zephyr repository gives an overview of the nRF5340 PDK support in Zephyr.
```

Comment:
I understand this is not yet in zephyr?

Reply:
Yes, that's why there's no reference. :(